src/main/java/com/minecolonies/coremod/entity/ai/citizen/builder/EntityAIStructureBuilder.java

```diff
@@ -570,7 +571,6 @@ private AIState structureStep()
         worker.faceBlock(job.getStructure().getBlockPosition());
 
 
-
         final BlockPos coordinates = job.getStructure().getBlockPosition();
         @Nullable final Block block = job.getStructure().getBlock();
         @Nullable final IBlockState blockState = job.getStructure().getBlockState();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the declaration of "blockState" closer to the code that uses it. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1941)